File: library/src/main/java/com/yarolegovich/slidingrootnav/util/SideNavUtils.java
package com.yarolegovich.slidingrootnav.util;

/**
 * Created by yarolegovich on 25.03.2017.
 */

public abstract class SideNavUtils {

    public static float evaluate(float fraction, float startValue, float endValue) {
        return startValue + fraction * (endValue - startValue);
    }
}

File: library/src/main/java/com/yarolegovich/slidingrootnav/util/SideNavUtils.kt
package com.yarolegovich.slidingrootnav.util

/**
 * Created by yarolegovich on 25.03.2017.
 */

object SideNavUtils {

    fun evaluate(fraction: Float, startValue: Float, endValue: Float): Float {
        return startValue + fraction * (endValue - startValue)
    }
}
